selfdrive/car/subaru/carstate.py

```diff
@@ -248,7 +290,7 @@ def get_cam_can_parser(CP):
       ]
     else:
       signals = [
-        ("Counter", "ES_DashStatus"),
+        ("COUNTER", "ES_DashStatus"),
```

Comment:
feel free to do these fixes in another PR. btw, the CANParser already adds `COUNTER` and `CHECKSUM` signals if present, so not really needed.

Reply:
ok, I'll test and make PR

Reply:
Done: https://github.com/commaai/openpilot/pull/27946